components/resources/brave_components_strings.grd

```diff
@@ -549,10 +549,19 @@
 
       <message name="IDS_BRAVE_REWARDS_CONNECT_WALLET_CUSTODIAL_HEADER" desc="">Custodial accounts (requires ID verification)</message>
       <message name="IDS_BRAVE_REWARDS_CONNECT_WALLET_CUSTODIAL_TOOLTIP" desc="">Custodial accounts are accounts with trusted third-party services that hold your BAT for you.</message>
+      <message name="IDS_BRAVE_REWARDS_CONNECT_WALLET_SELF_CUSTODY_HEADER" desc="">Self-custody</message>
+      <message name="IDS_BRAVE_REWARDS_CONNECT_WALLET_SELF_CUSTODY_TOOLTIP" desc="">Self-custody gives you exclusive control over your BAT, along with complete responsibility for its security.</message>
```

Comment:
Not sure the comma is needed here.

Reply:
I think we can leave comma for now. String with comma was looked at specifically by copywriters too :)